### PR TITLE
feat(fluentd): use official fluent/fluentd:1.3 docker image

### PR DESCRIPTION
**Pull request by duanhongyi**
1、use official fluent/fluentd:1.3 docker image
2、reorganize directory structure according to fluent/fluentd:1.3 image
3、upgrade to the latest fluent-plugin
4、add list namespaces role to `logger-fluentd-clusterrole.yaml`

### CODE REVIEW

**Review by till — APPROVED** [review]

**Comment by Cryptophobia** [comment]
@duanhongyi, I'm ready to merge this in. Somehow missed it when reviewing previous PRs. Is this ready for merge - has it been tested or does it need some more testing?

**Review by till — APPROVED** [review]

File: charts/fluentd/templates/logger-fluentd-clusterrole.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ metadata:
     heritage: deis
 rules:
 - apiGroups: [""]
-  resources: ["pods"]
+  resources: ["pods", "namespaces"]
```
Comment (till) [inline charts/fluentd/templates/logger-fluentd-clusterrole.yaml:12]: 🎉

Comment (Cryptophobia) [reply charts/fluentd/templates/logger-fluentd-clusterrole.yaml:12]: Can someone @till @duanhongyi tell me why this is a necessary change?

**Review by Cryptophobia — APPROVED** [review]
lgtm! :+1:

**Comment by Cryptophobia** [comment]
Merging!

**Pull request merged**